### PR TITLE
Clean Styling Imports

### PR DESCRIPTION
**Pull request by ZaidKhan144**
### Link to Relevant Issue

This pull request resolves [#115](https://github.com/CouncilDataProject/cdp-frontend/issues/115)

### Description of Changes

- Imported cdp design, Mozilla protocol and Semantic UI react styles in `.storybook/preview.js` and `src/app/App.tsx` 
- Removed cdp design, Mozilla protocol and Semantic UI react styles imports from multiple components

### CODE REVIEW

**Review by JacksonMaxfield — APPROVED** [review]
Thanks for these changes @ZaidKhan144. Really appreciate it. I will let @tohuynh handle second approval and merge if all tests pass.

**Comment by ZaidKhan144** [comment]
@JacksonMaxfield Okay

**Review by tohuynh — CHANGES_REQUESTED** [review]
Thanks for getting to this. This looks good. I have three minor asks in the comments.

Could you remove the Mozilla protocol style from `src/components/Layout/TabbedContainer/Tab` and  `src/containers/PersonContainer/PersonContainer` as well?

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import { themes } from "@storybook/theming";
 import { StaticRouter } from 'react-router-dom';
+import "@councildataproject/cdp-design/dist/images.css";
+import "@mozilla-protocol/core/protocol/css/protocol.css";
 import "semantic-ui-css/semantic.min.css";
```
[the remark this answers is not in the document]
Comment (ZaidKhan144) [reply .storybook/preview.js]: fixed

File: src/components/Filters/SelectTextFilterOptions/SelectTextFilterOptions.tsx [hunk 2]
```diff
@@ -4,7 +4,6 @@ import { FilterState } from "../reducer";
 
 import isSubstring from "../../../utils/isSubstring";
 
-import "@mozilla-protocol/core/protocol/css/protocol.css";
 import "@mozilla-protocol/core/protocol/css/protocol-components.css";
```
[the remark this answers is not in the document]
Comment (ZaidKhan144) [reply src/components/Filters/SelectTextFilterOptions/SelectTextFilterOptions.tsx]: fixed

**Review by tohuynh — APPROVED** [review]
Thanks for making the changes.

**Pull request merged**